src/formatting_stack/strategies.clj

```diff
@@ -64,9 +80,12 @@
   The diff is compared against the `:target-branch` option."
   [& {:keys [target-branch impl files blacklist]
       :or   {target-branch "master"
-             impl          (impl/file-entries git-command "diff" "--name-only" target-branch)
+             impl          (impl/file-entries git-command "diff" "--name-status" target-branch)
              blacklist     (git-not-completely-staged :files [])}}]
   (->> impl
+       (remove (fn [s]
+                 (-> s (string/starts-with? "D\t"))))
```

Comment:
We should use `git-status/deleted-file?` here

Reply:
> We should use git-status/deleted-file? here

Nope, note `git-status`' ns docstring

Reply:
ouch, okay :P